src/main/java/org/minions/devfund/aldo/WhackAMole.java

```diff
@@ -0,0 +1,130 @@
+package org.minions.devfund.aldo;
+
+/**
+ * This class handle the WhackAMole game.
+ */
+class WhackAMole {
+    private static final int SCORE = 10;
+    private int score;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "score" to prevent any misunderstanding/clash with field "SCORE" defined on line 7. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)